server/src/Persistence.hs

```diff
@@ -23,6 +27,7 @@ import           Data.Foldable
 import           Data.Text                  (Text)
 import qualified Data.Text                  as Text
 import           Data.Traversable
+import           Database.SQLite.Simple
```

Comment:
It would be nice to use a qualified import; for function like `execute`, `open`, and `field`, it is not immediately obvious that they relate to SQLite at the call site.